search/setup.cfg

```diff
@@ -11,6 +11,10 @@ max-line-length = 120
 # modify --cov-fail-under parameter after adding unit tests
 [tool:pytest]
 addopts = --cov=search_service --cov-fail-under=0 --cov-report=term-missing:skip-covered --cov-report=xml --cov-report=html -vvv
+env =
+    AWS_REGION=eu-west-1
+    AWS_ACCESS_KEY_ID=foo
```

Comment:
why do we need this? manual doesn't mention those vars so should they really be required for tests?

Reply:
`Session().get_credentials()` looks for them in [a couple of places](https://boto3.amazonaws.com/v1/documentation/api/latest/guide/credentials.html#configuring-credentials) and env variables are the easiest to mock for it not to complain.